samples/scala-value-entity-customer-registry/src/main/resources/logback.xml

```diff
@@ -0,0 +1,13 @@
+<configuration>
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
```

Comment:
this should be changed to json, same as we have in the java samples, but could be done separately after #496

Reply:
done